dev/bots/test.dart

```diff
@@ -295,8 +297,6 @@ Future<void> _runWebToolTests() async {
     path.join(flutterRoot, 'packages', 'flutter_tools'),
     forceSingleCore: true,
     testPaths: <String>[path.join('test', 'web.shard')],
-    enableFlutterToolAsserts: true,
```

Comment:
why removing the asserts?

Reply:
it defaults to true

Reply:
i'm actually surprised this didn't get flagged by the analyzer...

Reply:
ah we haven't yet enabled avoid_redundant_argument_values